scripts/build_mirror_config.sh

```diff
@@ -5,6 +5,10 @@ CONFIG_PATH=$1
 export ARCHITECTURE=$2
 export DISTRIBUTION=$3
 
+# Handling default
+[[ -z $APT_RETRIES_COUNT ]] && APT_RETRIES_COUNT=20
+export APT_RETRIES_COUNT
```

Comment:
>APT_RETRIES_COUNT [](http://example.com/codeflow?start=7&length=17)

Will you reuse this constant for `curl`, etc?

Reply:
No. It is for `apt` only